### PR TITLE
Clean up --json everywhere

### PR DESCRIPTION
**Pull request by krzyzacy**
after https://github.com/kubernetes/test-infra/pull/3777, and deploy pull-bazel-test-infra

ref https://github.com/kubernetes/test-infra/issues/3365

### CODE REVIEW

**Comment by fejta** [comment]
Very eager to see this one get in! 😀

**Comment by krzyzacy** [comment]
/assign @fejta 

Moved `fake-*` scripts to under `jenkins/` dir for now. (Since they are two liners :-( )

**Comment by krzyzacy** [comment]
cc @cjwagner this has a non-zero chance to break something somewhere :-)

File: jenkins/bootstrap_test.py [hunk 1]
```diff
@@ -1534,7 +1520,7 @@ def check(job, name):
             return job_name
 
         self.check_bootstrap_yaml('job-configs/kubernetes-jenkins/bootstrap-maintenance-ci.yaml',
```
Comment (krzyzacy) [inline jenkins/bootstrap_test.py]: done

File: jenkins/bootstrap_test.py [hunk 2]
```diff
@@ -1609,7 +1595,7 @@ def check(job, name):
 
         self.check_bootstrap_yaml(
             'job-configs/kubernetes-jenkins/bootstrap-ci.yaml',
-            check, use_json=True)
+            check)
```
Comment (krzyzacy) [inline jenkins/bootstrap_test.py]: done

**Comment by krzyzacy** [comment]
things will be red for ~5min till prow picks up the new config

**Comment by krzyzacy** [comment]
okay seems fine now :-)

**Pull request merged**